### DISCUSSION TITLE
Rendering two cameras into the same window.

### DISCUSSION THREAD

**Discussion by Domattee** [google-deepmind/mujoco#50] [Asking for Help] [answered]
I have resolved this issue, turns out that reading the documentation thoroughly is helpful.

I misinterpreted the mjRect Parameters width and height as endpoints on a canvas, when they are actually deltas!

**Comment by yuvaltassa** [comment, marked as answer]
Happy to hear. BTW, MuJoCo supports [side-by-side stereo rendering](https://github.com/deepmind/mujoco/blob/main/include/mjvisualize.h#L146), if that's what you're after. Both [modeled cameras](https://github.com/deepmind/mujoco/blob/main/include/mjmodel.h#L742) and the [free camera](https://github.com/deepmind/mujoco/blob/main/include/mjmodel.h#L457) have an Inter Pupillary Distance that determines the separation of the two "eyes". You should be able to set [this enum](https://github.com/deepmind/mujoco/blob/main/include/mjvisualize.h#L291) and see this. Perhaps we should expose this option in the `simulate` UI? But it's rarely useful in an interactive setting.
